spec/models/person_spec.rb

```diff
@@ -165,4 +165,19 @@
       expect { create(:person, :pre1900) }.to raise_exception(ActiveRecord::RecordInvalid, 'Validation failed: Birth date must be after 1900-01-01.')
     end
   end
+
+  describe '#update_nomis_data' do
+    let(:import_alerts_and_personal_care_needs) { instance_double('Profiles::ImportAlertsAndPersonalCareNeeds', call: true) }
+
+    before do
+      allow(Profiles::ImportAlertsAndPersonalCareNeeds).to receive(:new).and_return(import_alerts_and_personal_care_needs)
```

Comment:
this statement `allow(Profiles::ImportAlertsAndPersonalCareNeeds)...`  isn't required anymore

Reply:
Doh